### PR TITLE
Add Integration Menu

### LINKED ISSUE #60: Add Integration Page Menu

### PR DESCRIPTION
**Pull request by Dogacel**
Also careful about the react-router. I have updated it to v6 and I think the syntax is much better. For example we are not using history.push instead we use a new hook called navigate.

Closing #60 

image

Open to design ideas / suggestions.

### CODE REVIEW

File: web/src/views/IntegrationsView.jsx [hunk 1]
```diff
@@ -1,171 +1,77 @@
...
+              <Link to="shopify">Shopify</Link>
+            </Menu.Item>
+          </SubMenu>
+          <SubMenu
+            key="sub4"
+            icon={<SettingOutlined />}
+            title="Custom Integrations"
+          >
+            <SubMenu key="sub3" title="Submenu">
+              <Menu.Item key="7">Option 7</Menu.Item>
```
Comment (AtaAtasoy) [inline web/src/views/IntegrationsView.jsx:56]: Do you have an idea on what could these options be ? Just curious 😃

**Review by AtaAtasoy — APPROVED** [review]
The code looks good. Maybe wwe could wait for any other ideas that could come up before closing this PR.

File: web/src/views/IntegrationsView.jsx (hunk 1 above)
Comment (Dogacel) [reply web/src/views/IntegrationsView.jsx:56]: This should be a draft PR. We should elobrate on that together IMO

Comment (olcayakman) [reply web/src/views/IntegrationsView.jsx:56]: looks good, we can decide on the options for this menu in our next meeting!

**Pull request merged**